aws-assume-role-policy/main.tf

```diff
@@ -0,0 +1,48 @@
+data "aws_iam_policy_document" "assume-role" {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing version constraint for provider "aws" in "required_providers"